multiqc/modules/dedup/dedup.py

```diff
@@ -107,14 +121,29 @@ def dedup_general_stats_table(self):
             'scale': 'OrRd',
             'format': '{:,.2f}',
         }
+        headers['reads_removed'] = {
+            'title': 'Reads Removed',
+            'description': 'Non-unique reads removed after deduplication',
+            'min': 0,
+            'hidden': True
+        }
+        headers['mapped_after_dedup'] = {
+            'title': 'Post-DeDup Mapped Reads',
+            'description': 'Unique mapping reads after deduplication',
+            'min': 0,
+        }
         self.general_stats_addcols(self.dedup_data, headers)
 
     def dedup_alignment_plot (self):
         """ Make the HighCharts HTML to plot the duplication rates """
 
         # Specify the order of the different possible categories
         keys = OrderedDict()
-        keys['not_removed'] = { 'name': 'Not Removed' }
+        if 'mapped_after_dedup' in keys:
```

Comment:
You've just defined `keys` as an empty directory, so this will always be `False`..

Reply:
If you get rid of the `'hide_zero_cats': False` then I don't think you need this anyway - just define all categories and the ones that aren't used will just be ignored.

Reply:
Thanks for fixing this 🤦